versionedRouter.js

```diff
@@ -100,6 +104,37 @@ async function handleDest(ctx, version, destination) {
   ctx.set("apiVersion", API_VERSION);
 }
 
+async function routerHandleDest(ctx) {
+  const { destType, input } = ctx.request.body;
+  const routerDestHandler = getRouterDestHandler(destType);
+  const respList = [];
+  await Promise.all(
+    input.map(async input => {
+      try {
+        let respEvents = await routerDestHandler.processRouterDest(input);
```

Comment:
Before calling `processRouterDest`, you should check whether the method exists or not. If not, we should return a `404`.

Reply:
done.

Reply:
done